tests/main/parallel-install-classic/task.yaml

```diff
@@ -0,0 +1,86 @@
+summary: Checks parallel installation of classic snaps
+
+# core does not support classic confinement
+systems: [-ubuntu-core-*]
+
+prepare: |
+    # ensure we have no snap user data directories yet
+    rm -rf /home/test/snap
+    rm -rf /root/snap
+    rm -rf /var/snap/test-snapd-classic-confinement
+    rm -rf /var/snap/test-snapd-classic-confinement_foo
+
+    #shellcheck source=tests/lib/dirs.sh
+    . "$TESTSLIB"/dirs.sh
+    snap pack "$TESTSLIB/snaps/test-snapd-classic-confinement/"
+
+    case "$SPREAD_SYSTEM" in
+        fedora-*|arch-*|centos-*)
+            # although classic snaps do not work out of the box on fedora,
+            # we still want to verify if the basics do work if the user
+            # symlinks /snap to $SNAP_MOUNT_DIR themselves
+            ln -sf "$SNAP_MOUNT_DIR" /snap
+            ;;
+    esac
+
+    snap set system experimental.parallel-instances=true
+
+restore:
+    snap unset system experimental.parallel-instances
```

Comment:
I don't think this is necessary, as all of the snapd state should be wiped out in the restore-each, right?